Documentation/ApiOverview/CachingFramework/Index.rst

```diff
@@ -12,12 +12,19 @@ Caching in TYPO3
 ----------------
 
 TYPO3 uses multiple caching strategies to ensure fast content delivery. Depending
-on the content a page contains, TYPO3 chooses the best caching strategy for that use case.
+on the content a page contains, TYPO3 chooses the best strategy for the usage of caches.
+Each page has its own cache entry. There is no duplication of a cache between different pages. 
+However an extension developer can make an extension as uncached and use calls to the Caching Framework
+to do all cache handling. There are also cache options in TypoScript which may help to simplify this:
+t3tsref +++
+https://docs.typo3.org/m/typo3/reference-typoscript/10.4/en-us/Functions/Cache.html
 
 For example, you might have a fully-cacheable page, a page that is at least partially
 cacheable or a page that is completely dynamic. Dynamic elements in TYPO3 are also known
 as `USER_INT` or `COA_INT` objects - as these are the matching TypoScript objects used
-to render non-cacheable content.
+to render non-cacheable content. Or they are added in the file file:`localconf.php` of an
```

Comment:
to render non-cacheable content.

Reply:
BTW, are you aware of suggestions?

![image](https://user-images.githubusercontent.com/5037116/124098760-22777580-da5d-11eb-935c-cac73a0fd436.png)

These [can be applied as change directly](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).

Reply:
thanks @mbrodala , I missed that

Reply:
Why do you want to remove this information?

Reply:
Hope it's clear already, it's just moved in a separate paragraph (with some modification) ;-)